xclim/indices/run_length.py

```diff
@@ -259,6 +259,50 @@ def get_rl_stat(d):
     return rl_stat
 
 
+def rse(
+    da: xr.DataArray,
+    dim: str = "time",
+    index: str = "first",
+) -> xr.DataArray:
+    """Generate basic run summation function for positive values, containing accumulated values along runs.
+
+    Parameters
+    ----------
+    da : xr.DataArray
+        Input array.
+    dim : str
+        Dimension name.
+    index : {'first', 'last'}
+        If 'first' (default), the run length is indexed with the first element in the run.
+        If 'last', with the last element in the run.
+
+    Returns
+    -------
+    xr.DataArray
+        Values are 0 where da is False (out of runs).
+    """
+    da = da.astype(float)
+
```

Comment:
I think we could simply remove this line and use this as `rle` since this function is more general. Then we only apply the `int` conversion if data type is a bool
```suggestion
    """
    if da.dtype == bool: 
       da = da.astype(int)
```

Does that sound reasonable @aulemahal ?

Reply:
Yes that does absolutely make sense!

Reply:
Should I add these changes within this PR, or do you think it should be separate?

Reply:
In this PR it's all right. You can just modify `rle` 
1. replace the int conversion with the condition above
2. `da == 1` -> `da>0`
and remove `rse`